### PR TITLE
[MPIR-375] add plugin excludes feature for plugin-management report

### PR DESCRIPTION
**Pull request by belingueres**
Added pluginManagementExcludes list parameter.

### CODE REVIEW

**Comment by rfscholte** [comment]
Fixed in https://github.com/apache/maven-project-info-reports-plugin/commit/82f4bf23b79fe87d2e414c70ab2058e8f1b4146b
Thanks for the PR!

File: src/main/java/org/apache/maven/report/projectinfo/PluginManagementReport.java [hunk 1]
```diff
@@ -255,5 +283,41 @@ public int compare( Plugin a1, Plugin a2 )
...
+        /**
+         * Compares the given pattern against the given artifact. The pattern should follow the format
+         * <code>groupId:artifactId:type:classifier:version</code>.
+         * 
+         * @param pattern The pattern to compare the artifact with.
+         * @param artifact the artifact
+         * @return <code>true</code> if the artifact matches the pattern
+         */
+        protected boolean compareDependency( String pattern, Artifact artifact )
+        {
```
Comment (rfscholte) [inline src/main/java/org/apache/maven/report/projectinfo/PluginManagementReport.java]: Use org.apache.maven.shared.artifact.filter.PatternExcludesArtifactFilter instead

File: src/test/resources/plugin-configs/plugin-management-plugin-config-MPIR-375.xml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.apache.maven.plugin.projectinfo.tests</groupId>
+  <artifactId>plugin-management</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <packaging>jar</packaging>
+  <name>plugin management project info</name>
+  <dependencies>
```
Comment (rfscholte) [inline src/test/resources/plugin-configs/plugin-management-plugin-config-MPIR-375.xml]: Try to remove all unrequired stuff, like dependencies

File: src/main/java/org/apache/maven/report/projectinfo/PluginManagementReport.java (hunk 1 above)
Comment (belingueres) [reply src/main/java/org/apache/maven/report/projectinfo/PluginManagementReport.java]: I didn't knew that class...very handy!

**Comment by belingueres** [comment]
Pushed a new commit containing the suggested changes.

**Comment by rfscholte** [comment]
I've reopened this issue, so I can merge the improvements on the code. Can you first solve the conflicts?

**Comment by michael-o** [comment]
Merged with 82f4bf23b79fe87d2e414c70ab2058e8f1b4146b-

**Comment by jira-importer** [comment]
Resolve #228

**Comment by jira-importer** [comment]
Resolve #228

**Pull request closed without merging**